apis/core/lib/domain/dimension-mapping/update.ts

```diff
@@ -29,7 +34,10 @@ export async function update({
   dimensionMappings.changeSharedDimensions(sharedDimensions)
 
   dimensionMappings.onlyValidTerms = newMappings.onlyValidTerms
-  dimensionMappings.replaceEntries(newMappings.hadDictionaryMember)
+  const newEntries = dimensionMappings.replaceEntries(newMappings.hadDictionaryMember)
 
-  return dimensionMappings.pointer
+  return {
+    dimensionMapping: dimensionMappings.pointer,
+    hasChanges: newEntries.size > 0,
```

Comment:
One could maintain the number of entries to give better feedback to the user, 

'There are 3 new entries.'